cluster/gce/gci/configure-helper.sh

```diff
@@ -2041,6 +2041,11 @@ function start-fluentd-resource-update {
   wait-for-apiserver-and-update-fluentd ${fluentd_gcp_yaml} &
 }
 
+# Update {{ container-runtime }} with actual container runtime name.
+function update-container-runtime {
+  sed -i -e "s@{{ *container_runtime *}}@${CONTAINER_RUNTIME_NAME:-docker}@g" "$1"
```

Comment:
Please make a local variable from the argument, so it's much easier to read this code later (i.e. `local -r configmap_path="$1"`)

Reply:
Done.